File: app/src/main/java/co/tpcreative/supersafe/model/BreakInAlertsEntityModel.java
package co.tpcreative.supersafe.model;

import java.io.Serializable;

import co.tpcreative.supersafe.common.entities.BreakInAlertsEntity;

public class BreakInAlertsEntityModel implements Serializable {
    public int id;
    public String fileName;
    public long time;
    public String pin;
    public BreakInAlertsEntityModel(BreakInAlertsEntity index){
        this.id = index.id;
        this.fileName = index.fileName;
        this.time = index.time;
        this.pin = index.pin;
    }

    public BreakInAlertsEntityModel(BreakInAlertsModel index){
        this.id = index.id;
        this.fileName = index.fileName;
        this.time = index.time;
        this.pin = index.pin;
    }
}

File: app/src/main/java/co/tpcreative/supersafe/model/BreakInAlertsEntityModel.kt
package co.tpcreative.supersafe.model
import co.tpcreative.supersafe.common.entities.BreakInAlertsEntity
import java.io.Serializable

class BreakInAlertsEntityModel : Serializable {
    var id: Int
    var fileName: String?
    var time: Long
    var pin: String?

    constructor(index: BreakInAlertsEntity) {
        id = index.id
        fileName = index.fileName
        time = index.time
        pin = index.pin
    }

    constructor(index: BreakInAlertsModel) {
        id = index.id
        fileName = index.fileName
        time = index.time
        pin = index.pin
    }
}